Library/Formula/eg.rb

```diff
@@ -0,0 +1,25 @@
+class Eg < Formula
+  desc "Expert Guide. Norton Guide Reader For GNU/Linux"
+  homepage "https://github.com/davep/eg"
+  url "https://github.com/davep/eg/archive/eg-v1.01.tar.gz"
+  sha256 "bb7a2af37c8d5d07f785466692f21561257ff99106d2cb91db13ba2e946ff13b"
+  head "https://github.com/davep/eg.git"
+
+  depends_on "s-lang"
+
+  def install
+    inreplace "eglib.c", "/usr/share/", "#{HOMEBREW_PREFIX}/share/"
+    system "make"
+    bin.install "eg"
+    man1.install "eg.1"
```

Comment:
It failed when testing locally, but I'm quite new in build testing so maybe it was just me.